Fix invalid target naming

paulober opened the pull request.

will-v-pi reviewed (changes requested):
Can you add a quick comment to explain why, maybe "Remove files with special characters in filenames"?

paulober commented:
> Can you add a quick comment to explain why, maybe "Remove files with special characters in filenames"?

Sure. The second one I removed because it also contains a ref to a Russian domain, but I guess removing those refs is not something Rbpi supports. Therefor I now added the comment and removed the second line.

will-v-pi reviewed (approved):
Thanks, I'll merge this
